[QST] not able to find download cudf-0.15-SNAPSHOT-cuda10-1.jar jar

I am working on GPU implementation using Python on docker images and we are successfully completed that POC. But now we are trying implement the same using Spark 3.0 with Nvidia plugin and we are continuously following update on [https://github.com/NVIDIA/spark-rapids](https://github.com/NVIDIA/spark-rapids) but recently we found new update on https://github.com/NVIDIA/spark-rapids/blob/branch-0.2/docs/getting-started.md where got reference for **cudf-0.15-SNAPSHOT-cuda10-1.jar** jar but we are not able to find that jar anywhere. 

Can you please provide me some link from where we can download **cudf-0.15-SNAPSHOT-cuda10-1.jar** jar to proceed further on our POC.

We tried to create a build from https://github.com/NVIDIA/spark-rapids/tree/branch-0.1 using the following command.

**mvn package -Dmaven.test.skip=true**

But getting build error...

Downloading from snapshots: https://urm.nvidia.com:443/artifactory/sw-spark-maven/com/nvidia/rapids-4-spark-tests_2.12/0.1-SNAPSHOT/maven-metadata.xml
[WARNING] Could not transfer metadata com.nvidia:rapids-4-spark-tests_2.12:0.1-SNAPSHOT/maven-metadata.xml from/to snapshots (https://urm.nvidia.com:443/artifactory/sw-spark-maven): Transfer failed for https://urm.nvidia.com:443/artifactory/sw-spark-maven/com/nvidia/rapids-4-spark-tests_2.12/0.1-SNAPSHOT/maven-metadata.xml
[WARNING] Failure to transfer com.nvidia:rapids-4-spark-tests_2.12:0.1-SNAPSHOT/maven-metadata.xml from https://urm.nvidia.com:443/artifactory/sw-spark-maven was cached in the local repository, resolution will not be reattempted until the update interval of snapshots has elapsed or updates are forced. Original error: Could not transfer metadata com.nvidia:rapids-4-spark-tests_2.12:0.1-SNAPSHOT/maven-metadata.xml from/to snapshots (https://urm.nvidia.com:443/artifactory/sw-spark-maven): Transfer failed for https://urm.nvidia.com:443/artifactory/sw-spark-maven/com/nvidia/rapids-4-spark-tests_2.12/0.1-SNAPSHOT/maven-metadata.xml
Downloading from snapshots: https://urm.nvidia.com:443/artifactory/sw-spark-maven/com/nvidia/rapids-4-spark-tests_2.12/0.1-SNAPSHOT/rapids-4-spark-tests_2.12-0.1-SNAPSHOT-tests.jar
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for RAPIDS Accelerator for Apache Spark Root Project 0.1-SNAPSHOT:
[INFO] 
[INFO] RAPIDS Accelerator for Apache Spark Root Project ... SUCCESS [  2.642 s]
[INFO] RAPIDS Accelerator for Apache Spark SQL Plugin ..... SUCCESS [01:29 min]
[INFO] RAPIDS Accelerator for Apache Spark Shuffle Plugin . SUCCESS [ 10.152 s]
[INFO] RAPIDS Accelerator for Apache Spark Distribution ... SUCCESS [  7.095 s]
[INFO] RAPIDS Accelerator for Apache Spark Tests .......... SUCCESS [  5.362 s]
[INFO] rapids-4-spark-integration-tests_2.12 .............. FAILURE [  6.603 s]
[INFO] rapids-4-spark-api-validation ...................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:01 min
[INFO] Finished at: 2020-06-29T13:58:54+08:00
[INFO] ------------------------------------------------------------------------
**[ERROR] Failed to execute goal on project rapids-4-spark-integration-tests_2.12: Could not resolve dependencies for project com.nvidia:rapids-4-spark-integration-tests_2.12:jar:0.1-SNAPSHOT: Could not transfer artifact com.nvidia:rapids-4-spark-tests_2.12:jar:tests:0.1-SNAPSHOT from/to snapshots (https://urm.nvidia.com:443/artifactory/sw-spark-maven): Transfer failed for https://urm.nvidia.com:443/artifactory/sw-spark-maven/com/nvidia/rapids-4-spark-tests_2.12/0.1-SNAPSHOT/rapids-4-spark-tests_2.12-0.1-SNAPSHOT-tests.jar: Unknown host urm.nvidia.com -> [Help 1]**
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :rapids-4-spark-integration-tests_2.12

--- a/pom.xml
+++ b/pom.xml
@@ -534,6 +534,16 @@
     </build>
     <repositories>
         <repository>
+            <id>snapshots-repo</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+        <repository>
             <id>snapshots</id>
             <name>sw-spark-maven</name>
             <url>https://urm.nvidia.com:443/artifactory/sw-spark-maven</url>

